INZFS.MVC/Services/FileUpload/FileUploadService.cs

```diff
@@ -87,13 +84,20 @@ public async Task<string> Validate(IFormFile file)
             }
             
             //TODO : Switch to virus scanning service
-            var containsVirus = false; // await _virusScanService.ScanFile(file);
-            if (containsVirus)
+            var containsVirus = _virusScanService.ScanFile(file);
+            if (containsVirus == "Virus detected")
```

Comment:
Can't this return a boolean value instead of text ?

Reply:
Or is that if a string is returned, then the file has some issue, in which case the string can be simply returned as it.